other/install.php

```diff
@@ -2378,15 +2382,15 @@ function template_populate_database()
 	if (!empty($incontext['failures']))
 	{
 		echo '
-				<div style="color: red;">', $txt['error_db_queries'], '</div>
-				<ul>';
+		<div style="color: red;">', $txt['error_db_queries'], '</div>
```

Comment:
All main colors should be predefined in index.css, if not should be added (ignoring rest inline-coloring).